docs/overview/getting-started/virtual-machines.md

```diff
@@ -0,0 +1,94 @@
+# Virtual Machines
+
+Avalanche allows developers to implement their Dapp in the language of their choice using the 
+Virtual Machine (VM) framework. VMs define blockchains, and serve as application-level blueprints 
+for how blockchains are created.
+
+## Validators
+
+All Avalanche validators as members of the Avalanche primary network are required to run three VMs:
+
+- Coreth: Defines the Contract Chain (C-Chain); supports smart contracts written in Solidity.
+- Platform VM: Defines the Platform Chain (P-Chain); supports operations on staking and Subnets.
+- Avalanche VM: Defines the Exchange Chain (X-Chain); supports operations on Avalanche Native
+  Assets.
+
+Validators are able to install additional VMs on their node to validate additional Subnets in
+the Avalanche ecosystem. In exchange, validators receive staking rewards in the form of the Subnets'
+native gas tokens.
```

Comment:
This is not necessarily the subnets native gas token and some subnets are planning to separate out a staking reward token and the gas token. Could we make this statement more general?

Reply:
Yeah reworded this to be "reward token configured by Subnets".